lib/exq.ex

```diff
@@ -25,14 +26,14 @@ defmodule Exq do
   end
 
   def enqueue(pid, queue, worker, args) do
-    GenServer.call(pid, {:enqueue, queue, worker, args})
+    GenServer.call(pid, {:enqueue, queue, worker, args}, Config.get(:redis_timeout, 5000))
```

Comment:
If I add exq too the applications list in my mix.exs and Exq.enqueue(:exq, ...) will that be handled by the enqueuer? The problem I had was that redis timed out. so first I added the things in redis connection, and then I got a timeout from GenServer as it has 5 sec timeout (I believe) and when redis didn't timeout but took more than GenServers timeout it times out. This is how it looked to me I actually removed the timeout's in lib/exq.ex before submitting the PR as I thought they wasn't necessary but running without them for just 5 min (we have a really bad network situation or something at my job) there it was again. So I put it back. But as you said if it's the Enqueuer that later makes the call then I don't know how this might do any difference.

I'll try to remove it again and see if I can get some errors to paste..